Formula/getmail6.rb

```diff
@@ -0,0 +1,26 @@
+class Getmail6 < Formula
+  include Language::Python::Shebang
+
+  desc "Extensible mail retrieval system with POP3, IMAP4, SSL support"
+  homepage "https://github.com/getmail6/getmail6"
+  url "https://github.com/getmail6/getmail6/archive/refs/tags/v6.18.12.tar.gz"
+  sha256 "419dc11184b147ef4daac2ec9f136b1b37e6450e2f44099629a6b7650deb044a"
+  license "GPL-2.0-only"
+
+  depends_on "python@3.11"
```

Comment:
Does it need a specific version of Python?

Reply:
probably not, but I am specifying it mostly for being explicit.